explorer.md

```diff
@@ -2,3 +2,29 @@
 permalink: /explorer.html
 ---
 
+# Loc-I Explorer
+
+Test site:
+[http://loci-integration-app-test.s3-website-ap-southeast-2.amazonaws.com](http://loci-integration-app-test.s3-website-ap-southeast-2.amazonaws.com)
+
+## What
+
+A visual explorer that interfaces with the Loc-I reference Integrated Cache APIs. This tool will enable users to visualise and interactively explore (by clicking on links) and searching the graph of interconnected ontologies (concepts), Loc-I datasets, spatial features within them and linksets that describe spatial relationships between them (i.e. by location, text labels, codes). 
+     
+## Why
+
+The Loc-I integrated graph i.e. the datasets, spatial features within them, and links between them and to observation data stores that they reference, is a complex interconnected information space with billions of nodes and links between them. The Loc-I explorer provides users interfaces to enable to navigating of the graph. 
```

Comment:
either 'user interfaces' or 'users with interfaces' 

And instead of 'to enable to navigating of the graph' how about 'to navigate the graph'